utils.js

```diff
@@ -0,0 +1,9 @@
+module.exports.assert = {
+  equal: function (condition1, condition2, message) {
```

Comment:
let's rename `condition1` and `condition2` to `value1` and value2` since they are not actually conditional statements but values to be compared.